job-search-react/src/components/JobApp/JobApp.js

```diff
@@ -9,23 +9,25 @@ import Paginate from './Pagination/Pagination';
 
 export default function JobApp(props) {
 	const [jobData, setJobData] = useState(false);
-
+	const [clickEvent, setClickEvent] = useState(false);
 	const [formSubmitted, setFormSubmitted] = useState(false);
+	const getGrid = document.getElementById('jobgrid');
 
-	function formData(clickEvent) {
+	function formData(clickEvent, page = 1) {
```

Comment:
Please update the first argument here `clickEvent` to something like `e` or `event`. It confuses me a bit because it's named the same as the piece of state on line 12.